src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,46 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum Rank {
+
+	FIRST(6, 2_000_000_000),
+	SECOND(5, 1_500_000),
+	THIRD(4, 50_000),
+	FOURTH(3, 5_000),
+	NONE(-1, 0);
+
+	private int matchCount;
+	private int winningMoney;
+
+	Rank(int matchCount, int winningMoney) {
+		this.matchCount = matchCount;
+		this.winningMoney = winningMoney;
+	}
+
+	public static Rank findRankByMatchCount(int matchCount) {
+		return Arrays.stream(Rank.values())
+				.filter(rank -> rank.matchCount == matchCount)
+				.findFirst()
+				.orElse(NONE);
+	}
+
+	public int getMatchCount() {
+		return matchCount;
+	}
+
+	public int getWinningMoney() {
+		return winningMoney;
+	}
+
+	public long multiplyWinningMoney(int rankCount) {
+		return this.winningMoney * rankCount;
+	}
+
+	public boolean isGreaterThan(Rank rank) {
+		if (this.matchCount > rank.getMatchCount()) {
+			return true;
+		}
+		return false;
+	}
```

Comment:
다음과 같이 바로 return 해줘도 좋을 것 같아요 😄 
```suggestion
	public boolean isGreaterThan(Rank rank) {
		return this.matchCount > rank.getMatchCount();
	}
```

Reply:
아 그러네요! 앞으론 좀더 맑은정신에 코딩을 해야겟어요 ㅠㅠ